202312-quarto-reports/2023-12-quarto-reports.qmd

```diff
@@ -0,0 +1,278 @@
+---
+title: "Sharing research as Quarto reports"
+author: "Renata Diaz - December 6, 2023"
+format: uaz-revealjs
+editor: source
+---
+
+## Overview   {.smaller}
+
+| Topic                                       | Time          | Topic                              | Time          |
```

Comment:
This is very minor, so feel free to not change it for this workshop, but I think having the side by side rows with the schedule was confusing at first. I think having just one column each for "Topic" and "Time" or getting a line between the two sets would make it more clear.

Reply:
Updating this! (It didn't all fit at first, but it does now!)